PermissionsEx/permissions.yml

```diff
@@ -1329,7 +1329,81 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
-  beerus:
+ benten:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2B&fe&2n&010'
+      suffix: '&2'
+    permissions: []
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.benten
+        - essentials.fly
+        - essentials.speed.fly
```

Comment:
proof required of flying at very high speeds

this is missing from the powers list